### DISCUSSION TITLE
<Github page only shows readme.md file>

### DISCUSSION THREAD

**Discussion by AnitaNoemiSzabo** [community/community#127024] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi everyone,

I spent many hours to understand why only the README.md page is shown on the Github page. 

- The repository is here: https://[AnitaNoemiSzabo](https://your-username.github.io/AnitaNoemiSzabo/RestaurantFinder).github.io/RestaurantFinder
- The Github page is here: https://github.com/AnitaNoemiSzabo/RestaurantFinder.git 

Thank you very much in advance!

Best,
Anita

**Comment by davevad93** [comment, marked as answer]
Hi @AnitaNoemiSzabo ,

GitHub Pages looks for the `index.html` file in the root of your repository, otherwise it cannot render the page correctly, here's a structure example:

/your-repo
│
├── index.html
├── styles.css
└── README.md

If you need more info about setting up a GitHub Pages site, take a look at this from the docs:

1. [About GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)
2. [Creating a GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)

**Comment by AnitaNoemiSzabo** [comment]
Hi @davevad93 ,
Thank you for your help.
Best,
Anita

**Reply by davevad93** [reply]
You're welcome @AnitaNoemiSzabo , have a nice day!
